Package.swift

```diff
@@ -22,21 +22,31 @@ let package = Package(
     ],
     products: [
         .library(name: "SpeziStudy", targets: ["SpeziStudy"]),
+        .library(name: "SpeziLocalization", targets: ["SpeziLocalization"]),
         .library(name: "SpeziStudyDefinition", targets: ["SpeziStudyDefinition"])
     ],
     dependencies: [
         .package(url: "https://github.com/apple/FHIRModels.git", from: "0.7.0"),
         .package(url: "https://github.com/StanfordSpezi/Spezi.git", from: "1.8.1"),
-        .package(url: "https://github.com/StanfordSpezi/SpeziFoundation.git", from: "2.1.4"),
+        .package(url: "https://github.com/StanfordSpezi/SpeziFoundation.git", from: "2.2.0"),
         .package(url: "https://github.com/StanfordSpezi/SpeziHealthKit.git", from: "1.1.0"),
-        .package(url: "https://github.com/StanfordSpezi/SpeziScheduler.git", from: "1.2.7"),
+        .package(url: "https://github.com/StanfordSpezi/SpeziScheduler.git", revision: "9fb72378186681ccf465cac4f81c5302c58b7909"),
         .package(url: "https://github.com/StanfordSpezi/SpeziStorage.git", from: "2.1.0"),
-        .package(url: "https://github.com/apple/swift-collections.git", from: "1.1.4")
+        .package(url: "https://github.com/apple/swift-collections.git", from: "1.1.4"),
+        .package(url: "https://github.com/apple/swift-algorithms", from: "1.2.1")
     ] + swiftLintPackage(),
     targets: [
+        .target(
+            name: "SpeziLocalization",
```

Comment:
Nice addition 🚀

Reply:
@PSchmiedmayer the longer-term plan is to probably have this as a separate repo, but since it's not that much code atm, it's probably best to keep it in here for the time being.

we can always move it out w/out it being a breaking change, since SpeziStudy is still a pre-1.0 repo